src/csp_post_processor/__init__.py

```diff
@@ -1,7 +1,9 @@
+from .constants import NONCE_HTTP_HEADER
 from .processor import post_process_html
 
 __version__ = "0.1.0"
 
-__all__ = ["post_process_html"]
+__all__ = ["post_process_html", "default_headers"]
 
 default_app_config = "csp_post_processor.apps.CSPPostProcessConfig"
+default_headers = (NONCE_HTTP_HEADER,)
```

Comment:
there's no need for this indirection, make it explicit and just import the `NONCE_HTTP_HEADER` in the settings and add it to the `CORS_ALLOW_HEADERS` setting

Reply:
Thanks for the suggestion. I prefer this because it is more consistent with current settings and there's little indirection in exposing a variable from a package.

Reply:
we want settings to be as obvious as possible, especially when all the relevant code is under our control. When looking at the `base.py` module, I want to be able to see exactly what is in a setting without having to track down function calls or aliases or whatever. Please apply the change.

Reply:
> there's no need for this indirection, make it explicit and just import the `NONCE_HTTP_HEADER` in the settings and add it to the `CORS_ALLOW_HEADERS` setting

in `src/openforms/conf/base.py`:

```py
# imports at the top
from csp_post_processor.constants import NONCE_HTTP_HEADER

...

# CORS settings section
CORS_ALLOW_HEADERS = (
    list(default_cors_headers)
    + [NONCE_HTTP_HEADER]
    + config("CORS_EXTRA_ALLOW_HEADERS", split=True, default=[])
)
```